packages/terra-application-links/tests/wdio/application-tabs-spec.js

```diff
@@ -25,7 +25,7 @@ describe('ApplicationTabs', () => {
         '--terra-application-tabs-color': '#000',
         '--terra-application-tabs-font-size': '1rem',
         '--terra-application-tabs-line-height': '1',
-        '--terra-application-tabs-padding': '1rem 2rem',
```

Comment:
We should probably remove the `--terra-application-tabs-padding` and `--terra-application-tabs-spacer-width` reference screenshots

Reply:
ah, this is using `themeCombinationOfCustomProperties`, so I'd think we only need to regenerate this single screenshot